.github/workflows/release.yml

```diff
@@ -1,11 +1,32 @@
 name: Release
-'on':
-  release:
-    types:
-      - published
+on:
+  workflow_dispatch:
```

Comment:
Who is allowed to trigger this manually?

Reply:
Everyone with write access to the repo.